test/Engine/ProtoTest/ProtoTest.csproj

```diff
@@ -48,9 +16,12 @@
     <AssemblyOriginatorKeyFile>ProtoTest.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="nunit.framework, Version=2.6.3.13283, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
-      <HintPath>..\..\..\src\packages\NUnit.2.6.3\lib\nunit.framework.dll</HintPath>
-    </Reference>
+    <Compile Remove="ComponentTest\**" />
+    <EmbeddedResource Remove="ComponentTest\**" />
+    <None Remove="ComponentTest\**" />
```

Comment:
What's this?

Reply:
These were not added to the csproj before. Now I have to specifically remove them (since the default is to compile everythinig on disk)